apis/nucleus/src/components/listbox/ListBoxPopover.jsx

```diff
@@ -35,6 +35,7 @@ export default function ListBoxPopover({
   app,
   fieldName,
   stateName = '$',
+  autoFocus,
```

Comment:
```suggestion
  autoFocus = true,
```
and can remove the ?? below

Reply:
Yes, we can, but saw this comment after merge, sorry 😅